testing/jormungandr-testing-utils/src/testing/node/grpc/server/mod.rs

```diff
@@ -129,7 +129,17 @@ impl Node for JormungandrServerImpl {
         _request: tonic::Request<PeersRequest>,
     ) -> Result<tonic::Response<PeersResponse>, tonic::Status> {
         info!(method = %MethodType::GetPeers, "Get peers request received");
-        Ok(Response::new(PeersResponse::default()))
+        use bincode::Options;
+        let data = self.data.read().unwrap();
+        let mut self_gossip = Vec::new();
+        let config = bincode::options();
+        config.with_limit(512);
+        config
+            .serialize_into(&mut self_gossip, data.profile().gossip().as_ref())
+            .unwrap();
```

Comment:
The code duplication is because the gossip format is controlled by the poldercast crate and we decided not to expose it in chain-libs, correct?

Reply:
The gossip format is controlled by Jormungandr's `topology::gossip`, and that is not exposed outside of the node.